Server/mods/deathmatch/logic/CGame.cpp

```diff
@@ -3692,6 +3693,23 @@ void CGame::Packet_Vehicle_InOut(CVehicleInOutPacket& Packet)
                                         CVehicleInOutPacket JackedReply(pJacked->GetID(), VehicleID, 0, VEHICLE_NOTIFY_OUT_RETURN);
                                         m_pPlayerManager->BroadcastOnlyJoined(JackedReply);
 
+                                        CLuaArguments Arguments;
+                                        Arguments.PushElement(pPed);                     // player / ped
+                                        Arguments.PushNumber(ucOccupiedSeat);            // seat
+                                        Arguments.PushElement(pJacked);                  // jacked
```

Comment:
Arg 1 should be the player that left the vehicle (i.e. the jacked player `pJacked`?)
Arg 3 should be the player who pulled the occupant out of the vehicle (i.e. the jacker `pPed`?)

Reply:
Also, I guess that means arg 1 should be `pJacked` when calling `"onVehicleExit"`

Reply:
Fixed